server/server.go

```diff
@@ -747,6 +763,39 @@ func (r *rodeServer) UpdatePolicy(ctx context.Context, updatePolicyRequest *pb.U
 	return policy, nil
 }
 
+func (r *rodeServer) createIndex(ctx context.Context, indexName string) error {
```

Comment:
and this! This is great

Reply:
There's a similar function in `grafeas-elasticsearch` that also handles applying the mappings from a file and aliases, but it's still tied to some of the specific index types in that project. But maybe down the line we could reference that instead.